webapp/channels/src/components/advanced_text_editor/advanced_text_editor.tsx

```diff
@@ -409,6 +410,13 @@ const AdvancedTextEditor = ({
         }
     }, [showPreview]);
 
+    // Add useEffect to focus textbox when selectedPostFocussedAt changes
```

Comment:
The `Add useEffect to` is redundant.
```suggestion
    // Focus textbox when selectedPostFocussedAt changes
```